src/kixi/datastore/schemastore/conformers.clj

```diff
@@ -2,40 +2,49 @@
   (:require [clojure.core :exclude [integer? double? set?]]
             [clojure.spec :as s]))
 
+(defn double->int
+  "1.0 will convert to 1; anything else will be rejected"
+  [d]
+  (let [int-val (int d)]
+    (when (== int-val d)
+      int-val)))
+
+(defn double-str?
+  [^String s]
+  (not= -1 (.indexOf s ".")))
```

Comment:
Aha, it didn't, but making this new stuff keep that rule was a pita. Can get rid of some more bits if we agree on that.